crates/wasmparser/src/validator/operators.rs

```diff
@@ -90,29 +90,53 @@ struct Locals {
     all: Vec<(u32, ValType)>,
 }
 
+/// A Wasm control flow block on the control flow stack during Wasm validation.
+//
+// # Dev. Note
+//
 // This structure corresponds to `ctrl_frame` as specified at in the validation
 // appendix of the wasm spec
 #[derive(Debug)]
```

Comment:
Technically I do not see a reason why `Frame` could not also derive `Copy` and `Clone`.

Reply:
Seems reasonable to me to add yeah